NEWS.md

```diff
@@ -1,3 +1,14 @@
+flux-accounting version 0.21.0 - 2022-12-12
+-------------------------------------------
+
+#### Fixes
+
+* Specify `--all` option to `flux queue stop` in rc1 and sharness tests (#303)
```

Comment:
It might be better to just use the verbatim title from the PR:

>  * testsuite: stop all queues with --all option (#303)

Also an entry is missing for #302 
>  * testsuite: do not assume queues default to stopped (#302)

Since these are both testsuite-only fixes, you may want to list them under `#### Testsuite`, and likely after new Features since the new Features are most important (I leave that up to you though)

Reply:
Thanks, I really like that suggestion. Sorry about missing the other PR! I just force-pushed a fix to add the other fix for #302, as well as move both of these to a separate `#### Testsuite` section under the `#### Features` section.

Reply:
We have a script in `flux-core` that can help you generate release notes, e.g.

```
 grondo@asp:~/git/flux-accounting$ ../flux-core.git/src/test/relnotes.sh 
Changes since v0.20.1:
 * edit-bank: add ability to edit parent bank (#299)
 * testsuite: do not assume queues default to stopped (#302)
 * github: change Ubuntu version for python-lint check (#300)
 * add-user: add bank validation (#277)
```

In case it ever comes in useful!

Reply:
Oh awesome! I had no idea. I'll definitely have to steal this for future releases ;)